_includes/changelog-table.html

```diff
@@ -1,6 +1,6 @@
-
 {% assign col1Title = 'Date' %}
 {% assign col1Classes = 'tablet:grid-col' %}
+{% assign col1ClassesMinimal = 'tablet:grid-col-3 desktop:grid-col-2' %}
```

Comment:
Added custom classes for the minimal (two-column) table display found in documentation page types. 

All other changes in this file are formatting fixes.